lib/remote-objects.js

```diff
@@ -577,6 +577,16 @@ function searchListenerTree(handlers, type, tree, i) {
   return listeners;
 }
 
+RemoteObjects.prototype._executeAuthorizationHook = function(ctx, cb) {
+  if(typeof this.authorization === 'function') {
+    this.authorization(ctx, cb);
+  } else {
+    process.nextTick(function() {
```

Comment:
Is this necessary? AFAIK, `process.nextTick` slows down server throughput, at least in benchmarks.